scripts/ansible-inventory.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+"""
+CDT-2221 Alpha Red Team
+Author: Anthony Swierkosz; ajs2576
+
+Updates local Ansible inventory with repository inventory.
+"""
+
+import shutil
+from os import path, makedirs
+
+
+def update_inventory():
+    """
+    Updates the local Ansible inventory file with the repository inventory.
+
+    :return: None
+    """
+    # Generate source path from current file location
+    base_path = path.dirname(__file__)
+    src_path = path.abspath(path.join(base_path, "..", "configs", "red-inventory.yaml"))
```

Comment:
Would be nice to make this more flexible. Current inventory is statically defined as `red-inventory.yaml`. Should make this an issue.

Reply:
Being address in https://github.com/aisgbnok/Ansible-2221/issues/7.